### ISSUE TITLE
Open discussions

### ISSUE THREAD

**Issue by tm1000** [bitfocus/companion-module-bss-soundweb#1]
Hey @AndyFug do you have the ability to open discussions or is that something only bitfocus can do? It took me a while to get the module working because of how (in my opinion) ugly Harmon makes this..

**Comment by AndyFug** [comment]
Hi,

I have actually opened a Discourse server for discussions (Since I had noticed other module maintainers had done a similar thing).  Here is an invite link (sorry... I haven't got round to putting it on the README yet):

[http://discourse.andrewfugle.co.uk/invites/4rZUvogb3F](http://discourse.andrewfugle.co.uk/invites/4rZUvogb3F)

I can't see the option to enable discusssions here, so I suspect I don't have that level of access.   That feels like a better solution... certainly in the long term.  I'll ask some of the Bitfocus maintainers for their thoughts.

Would be interesting to hear if/how you think we can make it any simpler or improve the Help to get people up and running faster.  Some screenshots would probably help I'm guessing.

**Comment by tm1000** [comment]
> Would be interesting to hear if/how you think we can make it any simpler or improve the Help to get people up and running faster. Some screenshots would probably help I'm guessing. 

The biggest problem I had was that I was spending about an hour trying to figure out why nothing was working with the right addresses, the connection showed connected. I opened wireshark and noticed there was no data flowing and I restarted the connection and everything started working. This is probably because I was going back and forth reconnecting my wireless network so Im not 100% sure this is a problem with your code or with companion itself.
